package.json

```diff
@@ -22,15 +22,17 @@
     "test": "jest --forceExit",
     "test:coverage": "jest --forceExit --coverage",
     "security_audit": "npx audit-ci --config audit-ci.json --skip-dev",
-    "int-test": "rm -rf cypress/downloads && cypress run",
-    "int-test-ui": "rm -rf cypress/downloads && cypress open",
+    "int-test": "rm -rf cypress/downloads && cypress run --browser chrome",
+    "int-test-ui": "rm -rf cypress/downloads && cypress open --browser chrome",
```

Comment:
It's not doing any harm being there, but the `--browser` switch won't have any effect locally as chrome is always the default anyway with `cypress open`